spec/models/team_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Team, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Let's install shoulda matchers and, after it, this kind of PR should specify the "logical schema" (i.e. validations, AR associations, etc.) in the model specs.